exercise_1/task_1.cpp

```diff
@@ -0,0 +1,39 @@
+#include <iostream>
+#include <string>
+#include <cctype>
+
+using namespace std;
+
+int main()
+{
+	string words;	
+	int vowels = 0, consonants = 0, others = 0;
+	
+	cout << "Enter words (q to quit)\n";
```

Comment:
If you want to be sure, the note is displayed, please use "std::endl" instead of "\n"